libs/application/templates/official-journal-of-iceland/src/lib/utils.ts

```diff
@@ -118,6 +130,11 @@ export const getSignatureDefaultValues = (signature: any, index?: number) => {
   return { institution: signature.institution, date: signature.date }
 }
 
+export const isAddition = (
+  addition: unknown,
+): addition is z.infer<typeof additionSchema> =>
+  additionSchema.safeParse(addition).success
+
```

Comment:
Can you explain  the `unknwon` and then forcing into `z.infer<typeof additionSchema>` (also in other lines). 😊  just curious

Reply:
Við notum zod schemu til að validera fielda í umsóknum. Þarna er ég að athuga hvort gefið inntak sé af þessari schemu.